reconcile/terraform_cloudflare_resources.py

```diff
@@ -127,20 +152,201 @@ def build_clients(
     return clients
 
 
+def _build_oc_resources(
+    cloudflare_namespaces: Iterable[NamespaceV1],
+    secret_reader: SecretReaderBase,
+    use_jump_host: bool,
+    thread_pool_size: int,
+    internal: Optional[bool] = None,
+    account_names: Optional[Iterable[str]] = None,
+) -> tuple[ResourceInventory, OCMap]:
+    ri = ResourceInventory()
+
+    oc_map = init_oc_map_from_namespaces(
+        # The Namespace Protocol isn't happy with NamespaceV1, but since it's a
+        # generated class there isn't a lot that can be done here.
```

Comment:
I think there is `e2eTests` missing in disable to make protocol happy. See above

Reply:
Ah, different `Cluster`... completely missed that.

Reply:
Fixed